lib/core/versions/latest/Operation.ts

```diff
@@ -283,6 +283,7 @@ export default class Operation {
     // Verify operation specific payload schema.
     switch (operationType) {
       case OperationType.Create:
+        decodedPayload.didDocument = JSON.parse(Encoder.decodeAsString(decodedPayload.didDocument));
```

Comment:
May want to add a comment about what happens when JSON.parse fails... we should expect encoding attacks to be among the most common out of the gate.

Reply:
Good point, I will make sure we do that. This pr is for visibility into the changes we are making after integration test. Not final yet so feel free to comment on anything you see should change.

Reply:
We handle bad requests in the request handler so bad json will not cause anything dangerous. The request will be rejected.